docs/concepts/vega-protocol.md

```diff
@@ -2,69 +2,273 @@
+
+## Asset governance
+Details on asset governance to come. 
+
+<!--
+### New asset proposal [WIP]
+-->
+
+## Market governance
+Markets are proposed and voted into existence by Vega tokenholders. The parameters for a market all need to be defined in the proposal.
+
+Some market parameters can also be changed. They can only be proposed by a liquidity provider with enough equity-like stake in the market, and need to be voted for by a sufficient number of tokenholders and/or liquidity providers.
+
+When creating a market governance proposal, whether it is for a new market or to change parameters for an existing market, it's recommended that you sense check the proposal and share the final details with the tokenholder community before proposing, so that you can garner support and make any necessary amends. 
+
+Read more:
+* [Vega community forum](https://community.vega.xyz): Share your draft proposals for community discussion.
+* [New market proposal](../tutorials/proposals/new-market-proposal.md): Guide to submitting a proposal for a new market using the command line
+* [Update market proposal](../tutorials/proposals/update-market-proposal.md): Guide to submitting a proposal to change a market using the command line
+
+### Propose a new market
+Tokenholders can propose new markets, which then need to be voted on by other tokenholders. If the market proposal gets a majority of tokeholder support, then it will be enacted. The required majority is defined by the network parameter `governance.proposal.market.requiredMajority`. 
+
 :::info
-You'll need a Vega Wallet for staking and receiving rewards. Connect to wallets and see your account balance on the [Vega token website](https://token.vega.xyz). CoinList custodial users should confirm with CoinList how staking works for them.
+A liquidity commitment is optional when proposing a market, but a market will not enter into continuous trading until its liquidity needs are met.
+:::
+
+To propose a market, you'll need to provide the details required for the market to begin trading right away. While some of the fields are free-text, others are constrained by a range set through network parameters, to ensure that the values provided are fit for purpose.
+
+Required fields include: 
+* Instrument details, including a human-readable name, an understandable shortcode for the market, the type of product (futures)
+* Risk model parameters
+* Product specifics including the settlement asset and quote name
+* Decimal places for the settlement asset, market, and positions 
+* Oracle details, including the oracle's public key, specifications for the settlement price and trading termination, and data filters
+* Liquidity monitoring parameters, including the target stake parameters, triggering ratio and auction extension
+
+Optional fields include: 
+* Metadata so that people can easily interpret the market's details
```

Comment:
It is optional but we should strongly encourage this and also somehow have a resource to discover convetions for various type of standard and useful metadata. Not sure the best way to achieve this… maybe community team have been thinking about this (or should be) when they are thinking about the whole process...